### PR TITLE
Prevent buildup of threads in specs

### PR DESCRIPTION
**Pull request by cantino**
@dsander, I added

```ruby
config.after(:each) do |example| 
  sleep 1
  p example.metadata[:full_description]
  pp Thread.list.map { |t| t[:name] || t[:rufus_scheduler_work_thread] || t == Thread::main } 
end 
```

to my rails_helper.rb and saw a large number of lines like `"rufus_scheduler_70250065071700_scheduler"`.  The following changes got rid of them.

### CODE REVIEW

File: app/concerns/long_runnable.rb [hunk 1]
```diff
@@ -117,6 +121,7 @@ def boolify(value)
     end
 
     private
+
```
Comment (dsander) [inline app/concerns/long_runnable.rb]: The good old battle over the empty space after the `private` modifier :smile: Maybe we should link the style guide (github or bbatsov) which huginn follows in the contributing section?

File: lib/agent_runner.rb [hunk 2]
```diff
@@ -17,10 +17,12 @@ def initialize(options = {})
     @scheduler = Rufus::Scheduler.new(frequency: ENV['SCHEDULER_FREQUENCY'].presence || 0.3)
 
     @scheduler.every 5 do
+      p :every
```
Comment (dsander) [inline lib/agent_runner.rb]: Should those stay or are they leftovers?

**Comment by dsander** [comment]
Looks good to me otherwise.

Nice, that might also resolve some of the random spec failure we were getting.

File: lib/agent_runner.rb (hunk 2 above)
Comment (cantino) [inline lib/agent_runner.rb]: Oops, good catch.

File: app/concerns/long_runnable.rb (hunk 1 above)
Comment (cantino) [inline app/concerns/long_runnable.rb]: Haha, I don't feel strongly. Well, except apparently I do enough to keep changing it :)

**Comment by cantino** [comment]
Fixed merge with master.

File: app/concerns/long_runnable.rb (hunk 1 above)
Comment (dsander) [inline app/concerns/long_runnable.rb]: Heh, for me it's the other way around, I will try to remember it for huginn :)

**Comment by dsander** [comment]
:+1:

**Comment by cantino** [comment]
Thanks

**Pull request merged**